packages/logrotate/buildinfo.json

```diff
@@ -1,7 +1,7 @@
 {
   "single_source": {
     "kind": "url_extract",
-    "url": "https://fedorahosted.org/releases/l/o/logrotate/logrotate-3.9.1.tar.gz",
+    "url": "http://pkgs.fedoraproject.org/repo/pkgs/logrotate/logrotate-3.9.1.tar.gz/4492b145b6d542e4a2f41e77fa199ab0/logrotate-3.9.1.tar.gz",
```

Comment:
The pkgs.fedoraproject.org vhost has no https it seems. We're not transferring sensitive data and we're verifying the sha sum of the transferred data. https wouldn't improve/add anything for us.
The md5 hash is part of the url, so I'm assuming the Fedora project was thinking along the same lines when they decided not to offer https on this repo server.